tests/puppeteer/campaigns/data/demo/invoices.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   },
   OrderStatuses: {
     canceled: {id: 4, name: 'Canceled'},
-    paymentAccepted: {id: 9, name: 'Payment accepted'},
-    shipped: {id: 14, name: 'Shipped'},
+    paymentAccepted: {id: 7, name: 'Payment accepted'},
+    shipped: {id: 12, name: 'Shipped'},
```

Comment:
What happened here ? Do these ids change a lot ?

Reply:
No, first time and I don't know how :thinking:

Reply:
I think it should be interesting to ask the devs.
@PrestaShop/prestashop-core-developers do you know how some ids would change over night ?

Reply:
FYI: The id is the position of the status in this page 
![image](https://user-images.githubusercontent.com/48441421/80591921-d2e38700-8a1e-11ea-9ea6-334187a828fc.png)

Reply:
If it randomly (or not) change that's kinda annoying to deal with.

Reply:
@SimonGrn I don't see any merged PR in two last days with impact in Order Status :

https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+merged%3A%3E%3D2020-04-27+is%3Aclosed+milestone%3A1.7.7.0

Reply:
Could it be linked with the order of insertion in the database when installing? If yes that's a problem...

Reply:
We found it, it's a paypal status added when install checkout module.
So I think it's not safe to target by id (Since it's not a id). I will update my PR.

Reply:
Fixed